lms/djangoapps/branding/api.py

```diff
@@ -462,24 +464,21 @@ def _footer_logo_img(is_secure):
     # logo by default, so that it can display properly on edx.org -- both
     # within the LMS, and on the Drupal marketing site, which uses this API.
     try:
-        return _absolute_url_staticfile(is_secure, logo_name)
+        return _absolute_url_staticfile(is_secure, footer_logo_path)
```

Comment:
Question: Would `footer_logo_path` be removed if we remove `FOOTER_ORGANIZATION_IMAGE`, per above?

Reply:
No, `footer_logo_path` initialized from the `SiteConfiguration` model with the `'FOOTER_ORGANIZATION_IMAGE'` value. If that's not present, it would use `settings.FOOTER_ORGANIZATION_IMAGE`.

So to answer your question, if we remove `FOOTER_ORGANIZATION_IMAGE` we would still get a static path to footer-logo if the site configuration is present.